tidy3d/components/simulation.py

```diff
@@ -717,12 +717,15 @@ class Simulation(AbstractSimulation):
     .. TODO maybe resize?
     """
 
-    run_time: pydantic.PositiveFloat = pydantic.Field(
-        ...,
+    run_time: pydantic.PositiveFloat | RunTimeSpec = pydantic.Field(
+        RunTimeSpec(),
```

Comment:
eventually it would be nice to be able to just not specify anything for run_time but I think for the testing phase we can make it required and see how it goes.

Reply:
Yeah cause we do charge a minimum of 10% of estimated cost, so if the run time is often way overestimated, it may not be good for users.